test/service/test_response.py

```diff
@@ -1590,8 +1590,8 @@ def tearDownClass(cls):
 
     def _count_total_cells(self, response_json):
         """
-        Return the number of cell suspension inner entities and total cell count
-        per hit
+        Return per hit the number of cell suspension inner entities and sum
+        value of 'totalCells' from those cell suspension inner entities.
```

Comment:
I found the old wording more understandable, but I don't have a strong objection.